Macros/Calc_weights_from_blocks.sas

```diff
@@ -141,7 +142,16 @@
   
   run;
   
-  %File_info( data=&outlib..&out_ds, printobs=1000 )
+  %Finalize_data_set( 
+  data=&out_ds.,
+  out=&out_ds.,
+  outlib=&outlib.,
+  label="Weighting file, &geo1dlbl to &geo2dlbl",
+  sortby=&geo1. &geo2.,
+  restrictions=None,
+  revisions=Revised weights.,
```

Comment:
Perhaps better to add a _revisions=_ parameter to the _%Calc_weights_from_blocks()_ macro and then specify the _revisions=_ message when the macro is invoked in the main program. The value would then just get passed on through the _%Finalize_data_set()_ macro. That would be more transparent. 

Perhaps have a slightly more descriptive _revisions=_ message. You will need to enclose in _%str()_ if the message has any commas in it.